modules/Bio/EnsEMBL/Compara/Utils/Registry.pm

```diff
@@ -55,14 +55,83 @@ use strict;
 use warnings;
 
 use Bio::EnsEMBL::DBSQL::DBAdaptor;
+use Bio::EnsEMBL::DBSQL::DBConnection;
+use Bio::EnsEMBL::Utils::Argument qw(rearrange);
 use Bio::EnsEMBL::Compara::DBSQL::DBAdaptor;
 use Bio::EnsEMBL::Taxonomy::DBSQL::TaxonomyDBAdaptor;
 
+use constant SUFFIX_SEPARATOR => '__cut_here__';
+
 my %ports;
 my %rw_users;
 my %rw_passwords;
 
 
+=head2 load_previous_core_databases_if_needed
+
+  Arg[1]      : Integer $release_number
+  Example     : Bio::EnsEMBL::Compara::Utils::Registry::load_previous_core_databases_if_needed($current_release-1);
+  Description : Wrapper around load_previous_core_databases to only call it once
+  Returntype  : none
+  Exceptions  : none
+
+=cut
+
+my %loaded_previous;
+sub load_previous_core_databases_if_needed {
+    my ($release_number) = @_;
+    return unless defined &load_previous_core_databases;
```

Comment:
Wondering if we should issue at least a warning in this case? In what scenario are we happy to call this loading method, but allow it to silently not do anything?

Reply:
I made it this way to allow "old-style" production_reg_conf to coexist with my new-style file for Pan-Compara, but I can roll it out to all divisions and then Registry.pm can assume `load_previous_core_databases` always exists